docs/reference/nodes/iterator.md

```diff
@@ -0,0 +1,47 @@
+# Iteration Nodes
+
+Iteration Nodes are used to create and define loops within a workflow. 
+
+## Iterator Node
+The *Iterator* node is used to create a loop in a workflow. This node generates multiple execution subgraphs with the nodes following the *Iterator* node. Each execution of the loop uses one value from the input iterable object as if it was effectively iterating over its elements.
+
+![Iterator Node](../../images/nodes/iterator.png) 
+
+>[!NOTE]
+> *Iterator* Nodes can be nested, meaning that another *Iterator* node can be placed inside the loop of an *Iterator* node to achieve multiple levels of iteration.
+
+### Inputs
+
+- **inValues** : This plug of type [Object](../nodes/#plugs) expects an iterable Python object, such as lists, dictionaries, sets or tuples. The number of elements of the iterable object will determine the number of loops the *Iterator* node will execute.
+
+### Outputs
+- **outValue**: This plug of type [Object](../nodes/#plugs) will output one value of the input iterable object corresponding to the current iteration. This means that the output of this plug will change dynamically for each loop the iteration Node executes. 
+
+## IteratorEnd Node
+The *IteratorEnd* node is used to define the end of a loop started by an *Iterator* node. This means that all nodes between an *Iterator* and a *IteratorEnd* will be part of the execution loop. It is recommended to always have an *IteratorEnd* node per *Iterator* node in a workflow.
+
+![IteratorEnd Node](../../images/nodes/iteratorEnd.png)  
+
+>[!NOTE]
+> It is advised to connect to the *IteratorEnd* node the output generated in the loop via a custom `through` plug. 

```

Comment:
That all the data that must be outputted from the loop should go through the IteratorEnd operator. Otherwise, the IteratorEnd node will show up as dirty at the end of the execution. What the node does in it's execution is copying the values from the inputs to the outputs


![image](https://github.com/user-attachments/assets/7b389df6-8cc4-40dc-87ae-76f81b37e907)
VS
![image](https://github.com/user-attachments/assets/56d579c1-b020-4348-bddb-8f15d6c87686)

Reply:
Wow, i didn´t know that this was supported or designed in this way.
Then I have a question, whar value does the IteratorEnds returns? the value from the last iteration?

Reply:
Ideally you would accumulate the results of each iteration in whatever you pass through the iteratorEnd node so yes it will contain the value set at the last iteration.

Reply:
Thx for the explanation. Then, I would suggest to specify this in the Note. I think it's usefull to clarify that if you connect plugs in the IteratorEnd and you user the outputs latter, the value get outside the loop will be the value from the last iteration. What could be used for the ListAccumulator, if we want to add that.